frontend/tests/Factories/Course.php

```diff
@@ -56,6 +58,7 @@ public static function createCourse(
             'requests_user_information' => $requestsUserInformation,
             'show_scoreboard' => $showScoreboard,
             'needs_basic_information' => $needsBasicInformation,
+            'languages' => !is_null($languages) ? implode(',', $languages) : '',
```

Comment:
```suggestion
            'languages' => !is_null($languages) ? implode(',', $languages) : null,
```

recuerda que hay que evitar el uso de valores mágicos. `''` no es un lenguaje válido.

Reply:
Cierto